tests/fixtures/mod.rs

```diff
@@ -0,0 +1,56 @@
+use std::fs::File;
+use std::process::Command;
+use std::process::Stdio;
+use std::thread;
+use tiny_http::{Response, Server};
+
+const PAGE: &'static str = "tests/fixtures/index.html";
+
+pub fn spawn_local_http_server() {
+    let server = Server::http("0.0.0.0:8000").unwrap();
+    println!("Spawning http server");
+    thread::spawn(move || {
+        for request in server.incoming_requests() {
+            let response = Response::from_file(File::open(PAGE).unwrap());
+            request.respond(response).unwrap();
+        }
+    });
+}
+
+pub fn get_file_count(pattern: &str, dir: &str) -> usize {
```

Comment:
```suggestion
pub fn get_file_with_pattern_count(pattern: &str, dir: &str) -> usize {
```
Not good at naming things, but get_file_count is misleading

Reply:
✔️